lib/universal/git-hooks.ts

```diff
@@ -0,0 +1,108 @@
+
+import { parse } from "https://deno.land/std/flags/mod.ts";
+import { Command } from "https://deno.land/x/cliffy/command/mod.ts";
+
+async function initHooks() {
```

Comment:
add default args like:

`initHooks(gitHooksPath = path.join(Deno.cwd(), '.githooks') )`

Include anything else that might be optional.